web/styles/recent_view.css

```diff
@@ -375,11 +375,11 @@
     }
 
     .recent-view-topic-header {
-        width: 35%;
+        width: 32%;
```

Comment:
Total width of the `recent_topic_name` td element is 40%, so divided the topic and unread data headers in a ratio which is followed by the data row below.